README

```diff
@@ -60,13 +70,31 @@ add the required variables to the body section in the client ui
 Click 'SEND'
 
 
-Curl Examples
+Curl Examples:
 If the UI is not your thing, curl probably is! Here's an example of using
 curl to create a gateway node.
 
-curl --insecure --user admin:admin -d "ip_address=192.168.122.104" \
+curl --insecure --user admin:admin -d "ip_address=192.168.122.14" \
      -X PUT https://192.168.122.69:5000/api/gateway/rh7-gw2
 
+IPv6 Support:
+Make sure the IPv6 addresses used are global unicast ones, and then the
+URL link could be:
+
+curl --insecure --user admin:admin -d "ip_address=2006:ac81::1104" \
+     -X PUT https://[2006:ac81::1104]:5000/api/gateway/rh7-gw2
```

Comment:
In this example we are asking `rh7-gw1` to define a second gateway named `rh7-gw2` at that IP address, correct?  If so, should it be
```suggestion
     -X PUT https://[2006:ac81::1103]:5000/api/gateway/rh7-gw2
```
and similarly below?

Reply:
And it looks like the proper route is `/api/gateway/<target_iqn>/<gateway_name>` with both `target_iqn` and `gateway_name` required.

Can you test the updated examples to make sure they actually work?

Reply:
Without the `<target_iqn>` is the old style. I will copy this from my cmd history.

Reply:
> In this example we are asking `rh7-gw1` to define a second gateway named `rh7-gw2` at that IP address, correct? If so, should it be
> 

This is correct !

The IPv6 address `2006:ac81::1104` is in rh7-gw2 node. This command means it is requesting the `rh7-gw2` node to set up the gateway for `rh7-gw2` node.

> and similarly below?

Reply:
I'm confused.  If there is no gateway on `rh7-gw2` node yet, how can one request anything from it?

Reply:
> I'm confused. If there is no gateway on `rh7-gw2` node yet, how can one request anything from it?
> 

Checked the history commits about this change, for the old style the `<target_iqn>` was one argument instead. It seems someone made one mistake when changing the style, just deleted the `<target_iqn>` argument in the examples but forgot to add it in the URL path. 


> I'd expect `curl` connect to `rh7-gw1` (`::1103`) and provide `rh7-gw2` IP address (`::1104`) in `ip_address` argument. Sorry for being dense on a simple example but I want to understand what I am missing.

Okay, there is no limitation about which one node should be connected when creating gateways except the first gateway to be created must be the same node it is connecting to.